guardrails/src/drivers/rest/routers/guardrails.py

```
@@ -0,0 +1,121 @@
+                {
+                    "index": 0,
+                    "message": {"role": "assistant", "content": response_content},
+                    "finish_reason": "stop",
+                }
+            ],
+            "usage": {
+                "prompt_tokens": 0,
+                "completion_tokens": 0,
+                "total_tokens": 0,
+            },
+        }
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)